README.md

```diff
@@ -482,152 +491,143 @@ To make this work with one of your types, you only need to provide two methods:
 using nlohmann::json;
 
 namespace ns {
-void to_json(json& j, person const& p)
-{
-  j = json{{"name", p.name}, {"address", p.address}, {"age", p.age}};
-}
-
-void from_json(json const& j, person& p)
-{
-  p.name = j["name"].get<std::string>();
-  p.address = j["address"].get<std::string>();
-  p.age = j["age"].get<int>();
-}
+    void to_json(json& j, person const& p) {
+        j = json{{"name", p.name}, {"address", p.address}, {"age", p.age}};
+    }
+
+    void from_json(json const& j, person& p) {
+        p.name = j["name"].get<std::string>();
+        p.address = j["address"].get<std::string>();
+        p.age = j["age"].get<int>();
+    }
 } // namespace ns
 ```
 
-That's all. When calling the json constructor with your type, your custom `to_json` method will be automatically called.
+That's all! When calling the `json` constructor with your type, your custom `to_json` method will be automatically called.
 Likewise, when calling `get<your_type>()`, the `from_json` method will be called.
 
 Some important things:
 
 * Those methods **MUST** be in your type's namespace, or the library will not be able to locate them (in this example, they are in namespace `ns`, where `person` is defined).
-* When using `get<your_type>()`, `your_type` **MUST** be DefaultConstructible and CopyConstructible (There is a way to bypass those requirements described later)
+* When using `get<your_type>()`, `your_type` **MUST** be [DefaultConstructible](http://en.cppreference.com/w/cpp/concept/DefaultConstructible) and [CopyConstructible](http://en.cppreference.com/w/cpp/concept/CopyConstructible). (There is a way to bypass those requirements described later.)
```

Comment:
I removed the need for CopyConstructible. This way, some types can be used thanks to copy elision.
If you think this is not a really good idea, you could simply put that check back in the `static_assert`